apis/collectioncsvapi/collectioncsvapi.yaml

```diff
@@ -0,0 +1,493 @@
+                  status: success
+                  errmsg: null
+                responseCode: OK
+                result:
+                  response: SUCCESS
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/tocapiTOCDownload-Resource_Not_Found'
+              example:
+                id: api.collection.toc.upload
+                ver: v1
+                ts: '2020-11-27 15:20:24:393+0000'
+                params:
+                  resmsgid: null
+                  msgid: 30527359-4723-4b59-b4c0-0b5b14f9a3e2
+                  err: INVALID_COLLECTION_NAME
+                  status: INVALID_COLLECTION_NAME
+                  errmsg: Collection Name given in the file doesn’t match current Collection name. Please check and upload again.
+                responseCode: CLIENT_ERROR
+                result: {}
+        '404':
+          description: Resource Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/tocapiTOCDownload-Resource_Not_Found'
+              example:
+                id: api.collection.toc.upload
+                ver: v1
+                ts: '2020-11-27 15:27:56:969+0000'
+                params:
+                  resmsgid: null
+                  msgid: c148b326-7f46-4899-b0cb-31d563f225bc
+                  err: RESOURCE_NOT_FOUND
+                  status: RESOURCE_NOT_FOUND
+                  errmsg: rootId do_11316025545317580812 does not exist
+                responseCode: RESOURCE_NOT_FOUND
+                result: {}
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/tocapiTOCDownload-Resource_Not_Found'
+              example:
+                id: api.collection.toc.upload
+                ver: v1
+                ts: '2020-11-23 06:48:19:128+0000'
+                params:
+                  resmsgid: null
+                  msgid: 50f76a53-4710-4333-936f-c043031eec03
+                  err: SERVER_ERROR
+                  status: SERVER_ERROR
+                  errmsg: SERVER_ERROR
+                responseCode: SERVER_ERROR
+                result: {}
+      deprecated: false
+      security:
+        - bearer: []
+  '/collection/v1/toc/download/{collection-id}':
+    get:
+      tags:
+        - TOC APIs
+      summary: TOC Download
+      description: |-
+        This API is associated with download collection in csv format.
+
+        The endpoint for TOC download is /download
+        The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      operationId: TOCDownload
+      parameters:
+        - name: Content-Type
+          in: header
+          description: "The Content Type entity is the media type of the resource. Possible\
```

Comment:
@Jayaprakash8887 please refer to the earlier comment on the use of \ to mark line breaks. this is a global comment.